main.go

```diff
@@ -268,7 +402,10 @@ func main() {
 			Name:  "setup",
 			Usage: "Creates an initial auth token under new user \"admin\"",
 			Action: func(cctx *cli.Context) error {
-				db, err := setupDatabase(cctx)
+				cfg := newDefaultConfig(cctx)
+				cfg.Load(cctx.String("config"))
+				overrideSetOptions(cctx, cfg)
```

Comment:
I've moved out the 'new' but the Load needs to stay there - because we need cctx,String("config") in case the user has overridden the config file location.